x-pack/plugins/session_view/public/components/Process/index.tsx

```diff
@@ -22,12 +22,18 @@ interface IProcessDeps {
  * Renders a node on the process tree
  * TODO: as well as sections for tty output, alerts and file redirection.
  */
-function Process({ process, isSessionLeader = false, depth = 0, onProcessSelected }: IProcessDeps) {
+export function Process({
+  process,
+  isSessionLeader = false,
+  depth = 0,
+  onProcessSelected,
+}: IProcessDeps) {
   const { euiTheme } = useEuiTheme();
   const [childrenExpanded, setChildrenExpanded] = useState(isSessionLeader || process.autoExpand);
 
   useEffect(() => {
     setChildrenExpanded(isSessionLeader || process.autoExpand);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
hmm, should we not include the missing dependency? Which I image is isSessionLeader (the value for which should never change, but yea..)

I need to sort out my eslint ide plugin. so i can go over these. :)

Reply:
yeah, didn't want to touch the `useEffects` deps at this point, since it is not always wrong not having all the dependencies

Reply:
sounds good. i'll review these after i get my eslint working better.

Reply:
sure, if you use Vscode, I can help setting the Eslint, prettier, and lens if needed

Reply:
what I'm doing, for now, I added this script line `package-json`: 

"lint:session_view": "node scripts/eslint ./x-pack/plugins/session_view",

which only lints in the `session_view` plugin folder, and is super fast compared to running in all files

Reply:
I'm using lunar vim (neovim mod). I guess it probably just needs to know where the eslintrc file is.